CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Add 
+- [#216] Annotation for exposed ports
+
+### Removed
+- [#216] Exposing services and configmaps
```

Comment:
We didn't expose configmaps here. The configmap is just the location where the nginx-ingress reads configuration for non http ingress.